### PR TITLE
Fixing stuff

### PR DESCRIPTION
**Pull request by Lafaiet**

### CODE REVIEW

File: my_super_app.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 def say_something():
-    print("Hello git!!!")
+    print("Hello git kraken")
```
Comment (kalevkena) [inline my_super_app.py:3]: cool change

Comment (Lafaiet) [reply my_super_app.py:3]: Thanks @kalevkena

Comment (andreoja) [reply my_super_app.py:3]: Oh, I can comment :)

**Comment by froguez** [comment]
Hi, this new feature looks great, thank you!

**Pull request merged**